### PR TITLE
Moving to cobra for the CLI

### PR DESCRIPTION
**Pull request by arschles**
Fixes https://github.com/gomods/athens/issues/6

### CODE REVIEW

**Comment by bketelsen** [comment]
LGTM - don't know why there are conflicts in the readme?

**Comment by arschles** [comment]
I made some edits to the README to remove mentions of `vgoprox` directly on master (sorryyyyyy!)

**Comment by arschles** [comment]
@bketelsen ready for review

File: README.md [hunk 1]
```diff
@@ -37,30 +38,30 @@ code for it in the [./cli](./cli) directory, and build it with the following:
...
-- ... and it needs to have a `go.mod` file in its root
-- The go.mod file's 'module' directive must match `<module>`. `vgp` won't read that 
-value yet (that's planned though)
-- If there are any `vendor` directories under `<directory>`, they won't be ignored yet, but that's
-planned
+* It is hard coded to make requests against `http://localhost:3000`, so you'll need to have the
+  Athens server running to successfully use it (see [Development](#development) above)
+* `<directory>` will be zipped up and uploaded to the Athens server
+* ... and it needs to have a `go.mod` file in its root
+* The go.mod file's 'module' directive must match `<module>`. `vgp` won't read that
```
Comment (bketelsen) [inline README.md]: s/vgp/athens

**Comment by michalpristas** [comment]
I had a small issue here today with different behavior of go test for pre-1.9 version (not ignoring vendor for go test)
so I changed a script a bit to ignore it explicitly. other option would be to disallow 1.8 checks. I think it is fine. we target devs using vgo so at least 1.10. this would be my preferred option instead of tweaking script
@arschles @bketelsen thoughts?

also @bketelsen pls unblock if it is ok. I resolved your comments

**Comment by arschles** [comment]
@michalpristas I'm cool with just removing 1.8 checks!

Also, this needs a rebase

**Pull request merged**